packages/reframed/src/reframed.ts

```diff
@@ -147,6 +179,9 @@ async function reframeFromTarget(
 			if (scriptType) {
 				script.setAttribute("type", scriptType);
 			}
+			if (script.id === "qwikloader") {
+				script.defer = true;
```

Comment:
whoa, what is this about? why are we making qwikloader async? This is most likely wrong. Can you please explain what are you trying to achieve?

Reply:
sorry this was just part of debugging! I checked the code but missed this one! sorry this addition it totally unintentional! thanks for pointing this one out!! 🙏